### PR TITLE
Application upgrade: better texts in info modal

### PR DESCRIPTION
**Pull request by tibor-kulcar**

### CODE REVIEW

File: src/karmen_frontend/src/components/gateways/heartbeat.js [hunk 1]
```diff
@@ -46,24 +46,22 @@ const UpgradeModal = ({ shouldShow }) => {
     isOpen && (
       <Modal>
         <h1 className="modal-title text-center">
-          Application has been upgraded
+          Application has been upgraded, please reload the page to continue.
```
Comment (xaralis) [inline src/karmen_frontend/src/components/gateways/heartbeat.js:49]: Karmen has been updated, please reload the page to continue working.

File: src/karmen_frontend/src/components/gateways/heartbeat.js [hunk 2]
```diff
@@ -46,24 +46,22 @@ const UpgradeModal = ({ shouldShow }) => {
...
             onClick={() => {
               window.location.reload(true);
             }}
           >
-            Upgrade
+            Reload
           </button>
+
+          <small className="text-center">
+            If the button does not work,<br/> try pressing <code>Ctrl+R</code> or{" "}
```
Comment (xaralis) [inline src/karmen_frontend/src/components/gateways/heartbeat.js:62]: I do not understand this statement. How can it not work? This doesn't look like clean UX.

File: src/karmen_frontend/src/components/gateways/heartbeat.js [hunk 3]
```diff
@@ -46,24 +46,22 @@ const UpgradeModal = ({ shouldShow }) => {
...
-        </p>
         <div className="cta-box text-center">
           <button
             className="btn"
             onClick={() => {
               window.location.reload(true);
             }}
           >
-            Upgrade
+            Reload
```
Comment (xaralis) [inline src/karmen_frontend/src/components/gateways/heartbeat.js:58]: Reload now

File: src/karmen_frontend/src/components/gateways/heartbeat.js (hunk 1 above)
Comment (xaralis) [reply src/karmen_frontend/src/components/gateways/heartbeat.js:49]: General question (not sure if @tibor-kulcar can answer that): what if user had some work unfinished?

File: src/karmen_frontend/src/components/gateways/heartbeat.js (hunk 2 above)
Comment (tibor-kulcar) [reply src/karmen_frontend/src/components/gateways/heartbeat.js:62]: For example, whole application can somehowe become unresponsive.

File: src/karmen_frontend/src/components/gateways/heartbeat.js (hunk 1 above)
Comment (tibor-kulcar) [reply src/karmen_frontend/src/components/gateways/heartbeat.js]: ceebcf8e67434e732085bd85c934b9d42814969f

File: src/karmen_frontend/src/components/gateways/heartbeat.js (hunk 3 above)
Comment (tibor-kulcar) [reply src/karmen_frontend/src/components/gateways/heartbeat.js]: ceebcf8e67434e732085bd85c934b9d42814969f

File: src/karmen_frontend/src/components/gateways/heartbeat.js (hunk 2 above)
Comment (xaralis) [reply src/karmen_frontend/src/components/gateways/heartbeat.js]: OK, let's leave it this way.

**Pull request merged**